var/spack/repos/builtin/packages/py-morph-tool/package.py

```diff
@@ -13,27 +13,27 @@ class PyMorphTool(PythonPackage):
     git = "https://github.com/BlueBrain/morph-tool.git"
     pypi = "morph-tool/morph-tool-2.9.1.tar.gz"
 
-    version("develop", branch="master")
+    version("master", branch="master")
     version("2.9.1", sha256="305e9456c8047726588b23dfa070eb95ccbe5573e9fea3e0a83dc93eacdf61dc")
     version("2.9.0", sha256="c60d4010e17ddcc3f53c864c374fffee05713c8f8fd2ba4eed7706041ce1fa47")
 
-    variant("neuron", default=False, description="Enable additional neuron support")
-    variant("plotly", default=False, description="Enable additional plotly support")
+    variant("nrn", default=False, description="Enable additional neuron support")
```

Comment:
I didn't follow this, why did upstream want to have the variant as `nrn`?

Reply:
So it would match the extra name in the package's `setup.py`

Reply:
Ah, ok :)